mod/account/set/right/password/password_input.haml

```diff
@@ -0,0 +1,3 @@
+%input.form-control.d0-card-content#card_subcards___password_content{name: "card[subcards][+*password][content]", type: "password", placeholder: "Password"}
```

Comment:
This has the word `*password` hardcoded into it. That means if someone changes the name of the `*password` card, it will break.

If you change it from `+*password` to `+:password`, however, it should be fine.

I also think we can do without the weird id (`#card_subcards___password_content`)

Reply:
This would be a good time to bring up i18n (internationalization). Can you please use the configuration in `mod/account/config/locales` to make the "Password" placeholder language-specific?

Reply:
Where is this weird id coming from? It's the id of the input field by default.
Changing *password to :password breaks it (screenshot).
<img width="437" alt="Screenshot 2023-10-11 at 3 19 18 PM" src="https://github.com/decko-commons/decko/assets/44006691/df16adde-0a90-41ab-b9ec-2aa3f848bdde">